shotover/src/runner.rs

```diff
@@ -45,6 +45,10 @@ struct ConfigOpts {
 
     #[arg(long, value_enum, default_value = "human")]
     pub log_format: LogFormat,
+
+    //hot reloading functionality
+    #[clap(long)]
+    pub hotreload: bool, //added a new boolean flag which can be triggered using '--hotreload'
```

Comment:
`//added a new boolean flag which can be triggered using '--hotreload'` comments should be about what the code does, not how it was changed. Details about how the code is changed can be included in commit messages if desired.